### ISSUE TITLE
Support Contract Deployments with a Multisig

### ISSUE THREAD

**Issue by wottpal** [paritytech/contracts-ui#475]

**Comment by statictype** [comment]
what do you mean?

**Comment by wottpal** [comment]
Of course, sorry I should have done this in the first place: Professionally deployed smart contracts that have admin- or upgrade capabilities should not be governed or deployed by a single wallet, but instead a [multisig account](https://support.polkadot.network/support/solutions/articles/65000181826-how-to-create-and-use-a-multisig-account). Doing so is right now only supported by either using polkadot.js/apps or custom scripting as far as I know. 

The process looks like the following:
1. You initiate the deployment transaction on one machine (probably by passing a special flag/function to indicate that it's an "unfinished" multisig transaction.
2. Copy over the call-data hash to all necessary signatory machines and continue/finish the transaction there.

Would be nice to make this possible within contracts-ui as well.
